src/topology/algebra/monoid.lean

```diff
@@ -94,6 +94,17 @@ lemma filter.tendsto.mul_const (b : M) {c : M} {f : α → M} {l : filter α}
   (h : tendsto (λ (k:α), f k) l (𝓝 c)) : tendsto (λ (k:α), f k * b) l (𝓝 (c * b)) :=
 h.mul tendsto_const_nhds
 
+/-- Construct a unit from limits of units and their inverses. -/
+@[to_additive tendsto.add_units "Construct an additive unit from limits of additive units and their
```

Comment:
This probably creates `tendsto.add_units` instead of `filter.tendsto.add_units`.